Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/URLObject-2.4.3.egg-info'...
Listing './lib/python3.5/site-packages/ZenQueue-0.5.2.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/simplejson'...
Listing './lib/python3.5/site-packages/simplejson/tests'...
Listing './lib/python3.5/site-packages/simplejson-3.13.2.egg-info'...
Listing './lib/python3.5/site-packages/urlobject'...
Listing './lib/python3.5/site-packages/zenqueue'...
Listing './lib/python3.5/site-packages/zenqueue/client'...
Compiling './lib/python3.5/site-packages/zenqueue/client/benchmark.py'...
***   File "./lib/python3.5/site-packages/zenqueue/client/benchmark.py", line 57
    print 'Invalid address specified; defaulting to 127.0.0.1:3000'
                                                                  ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/zenqueue/client/common.py'...
***   File "./lib/python3.5/site-packages/zenqueue/client/common.py", line 31
    except ValueError, exc:
                     ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/zenqueue/client/http'...
Compiling './lib/python3.5/site-packages/zenqueue/client/http/async.py'...
***   File "./lib/python3.5/site-packages/zenqueue/client/http/async.py", line 15
    except httpc.ConnectionError, exc:
                                ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/zenqueue/client/http/sync.py'...
***   File "./lib/python3.5/site-packages/zenqueue/client/http/sync.py", line 17
    except urllib2.HTTPError, exc:
                            ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/zenqueue/client/native'...
Compiling './lib/python3.5/site-packages/zenqueue/client/native/common.py'...
***   File "./lib/python3.5/site-packages/zenqueue/client/native/common.py", line 51
    except Exception, exc:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/zenqueue/queue'...
Listing './lib/python3.5/site-packages/zenqueue/server'...
Compiling './lib/python3.5/site-packages/zenqueue/server/http.py'...
***   File "./lib/python3.5/site-packages/zenqueue/server/http.py", line 118
    except Exception, exc:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/zenqueue/server/native.py'...
***   File "./lib/python3.5/site-packages/zenqueue/server/native.py", line 81
    except socket.error, exc:
                       ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/zenqueue/utils'...
Listing './share'...
Listing './share/python-wheels'...
